pallets/subtensor/src/utils/misc.rs

```diff
@@ -248,6 +248,10 @@ impl<T: Config> Pallet<T> {
         BlockAtRegistration::<T>::get(netuid, neuron_uid)
     }
 
+    pub fn get_kappa_can_set_by_owner(netuid: NetUid) -> bool {
+        KappaCanSetByOwner::<T>::get(netuid)
+    }
+
```

Comment:
Not sure if this is needed, the code called inside the function is explicit enough?